tests/Blake2Spec.hs

```diff
@@ -24,8 +24,11 @@ blake2b_res = pack [
     ]
 
 -- parameter sets
-b2b_md_len :: [Int]; b2b_md_len = [ 20, 32, 48, 64 ];
-b2b_in_len :: [Int]; b2b_in_len = [ 0, 3, 128, 129, 255, 1024 ];
+b2b_md_len :: [Int]
+b2b_md_len = [ 20, 32, 48, 64 ]
+
+b2b_in_len :: [Int]
+b2b_in_len = [ 0, 3, 128, 129, 255, 1024 ]
```

Comment:
Aagh. You realize this was a straight-from-C port that benefits from resembling the original reference code?

Revert this entire file.

Reply:
I'm happy to revert these parameter set functions, I only changed them because the style wasn't idiomatic.

The actual `*_selftest` functions that were reworked were done so to avoid non-exhaustive pattern match warnings.